### ISSUE TITLE
The keys are a nightmare to use

### ISSUE THREAD

**Issue by laxdog** [sumonami/sumonami#4]
Whoops! Probably my bad for pushing this. 

Could still add controller support and get 4 extra players that way. Should probably strip the main game back to 4 players on keyboard and remap the keys to something useful. 

Or rework the keymap so it's easier with 8. Though I think that'll be hard.

**Comment by Stealthii** [comment]
Agreed - keyboard support should be a priority in the next release.
